montepy/input_parser/tally_parser.py

```diff
@@ -4,7 +4,14 @@
 
 
 class TallyParser(DataParser):
-    """ """
+    """
+    A barebone parser for parsing tallies before they are fully implemented.
+
+    .. versionadded:: 0.2.0
```

Comment:
Is this the correct version?

Reply:
Yes I made this a long time ago and checked through the git history.